src/core/apparmor-setup.c

```diff
@@ -0,0 +1,101 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+#if HAVE_APPARMOR
+#include <sys/apparmor.h>
```

Comment:
Maybe indent this with two spaces for readability: `#  include <sys/apparmor.h>`.

Reply:
@keszybz
Do you mean "#  include" or "  #include"? Should I do the same with the DEFINE_TRIVIAL_CLEANUP_FUNC... part?